updating/updating-restricted-network-cluster/index.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+[id="about-restricted-network-updates"]
+= About restricted network updates
+include::_attributes/common-attributes.adoc[]
+:context: about-restricted-network-updates
+
+toc::[]
+
+A restricted network environment is one in which your cluster nodes cannot access the internet. 
+For this reason, you must populate a registry with the installation images. 
+If your registry host cannot access both the internet and the cluster, you can mirror the images to a file system that is disconnected from that environment and then bring that host or removable media across that gap. 
+If the local container registry and the cluster are connected to the mirror registry's host, you can directly push the release images to the local registry.
+
+If multiple clusters are present within the restricted network, mirror the required release images to a single container image registry and use that registry to update all the clusters.
+
+== Performing a restricted network update
+
+You can use one of the following procedures to update a restricted network {product-title} cluster:
+
+* xref:restricted-network-update-OSUS.adoc#updating-restricted-network-cluster-OSUS[Restricted network updates using the OpenShift Update Service]
+
+* xref:restricted-network-update.adoc#updating-restricted-network-cluster[Restricted network updates using the `oc` command-line interface (CLI)]
```

Comment:
OC CLI can be used to update the cluster even if there is OSUS running for the disconnected environment.So the categorization is not correct .  Also `Restricted network updates using the OpenShift Update Service` might be misleading because connected clusters update using OpenShift update service. 

The right categorization would be 

1.  Updating disconnected environments when running OSUS in the disconnected environment
2.  Updating disconnected environments without running OSUS in the disconnected environment

Reply:
Sounds good. I was using the previous headings for these pages, so I'll update them to these titles or something similar.